README.md

```diff
@@ -1,33 +1,72 @@
 
-> Showcasing the power of [Tequila](https://partners.kiwi.com/presenting-tequila-revolution-travel-industry/)
+<hr />
+⭐️ If you like this project, please, star us on GitHub — it helps!
+## Demo
 
-This project is a living example of a client app you could start building with Tequila.
+Live demo: https://kiwicom-margarita.netlify.com (web version)
 
-- 🕸️ Web version: https://kiwicom-margarita.netlify.com
-- 🎮 GraphQL Playground https://p2kwd3i3a8.execute-api.eu-central-1.amazonaws.com/staging/graphql
-- 🚀 GraphQL Voyager: https://margarita-graphql-voyager.now.sh/
+#### Screenshots
+
+<img src="./docs/assets/mac.png" alt="Search" title="Search" width="100%"/>
 
 <div align="center">
-<img src="./docs/assets/search.png" alt="Search" title="Search" width="200"/>
-<img src="./docs/assets/results.png" alt="Results" title="Results" width="200"/>
-<img src="./docs/assets/bookings.png" alt="Bookings" title="Bookings" width="200"/>
+  <img src="./docs/assets/screens/search.png" alt="Search" title="Search" width="250"/>
+  <img src="./docs/assets/screens/results.png" alt="Results" title="Results" width="250"/>
+  <img src="./docs/assets/screens/mmb.png" alt="Bookings" title="Bookings" width="250"/>
 </div>
 
-## 📖 Documentation
+## Devstack
+
+This project follows react-native-web philosophy of **Write Once, Render Anywhere**.
+
+- React Native
+- Expo
+- React Native Web
+- Next.js
+- Relay
+- Flow
+
+More detailed info about our devstack can be found in [Technical Overview docs](https://margarita.kiwi.com).
+
+## App features
 
-The full documentation for **Margarita** can be found on our [website](https://kiwicom.github.io/margarita/docs/).
+App consists of 4 main parts which should cover whole app flow:
 
-## 💻 Developing
+- Search for connections
+- Booking
+- Manage my booking
+- Payments
 
-- [Getting Started](https://kiwicom.github.io/margarita/docs/getting-started)
-- [Technical Overview](https://kiwicom.github.io/margarita/docs/technical-overview)
+More detailed info can be found in [Features and Roadmap docs](https://kiwicom.github.io/margarita/docs/features-roadmap).
+
+## Documentation
+
+- **[Getting Started](https://kiwicom.github.io/margarita/docs/getting-started)**
+- [Tequila API](https://kiwicom.github.io/margarita/docs/tequila-api)
 - [Contributing](https://kiwicom.github.io/margarita/docs/contributing)
 
-## 📄 License
+Full documentation for **Margarita** can be found on [kiwicom.github.io](https://kiwicom.github.io/margarita/docs/).
+
+## How it works
+
+<img src="./docs/assets/schema.png" alt="Schema" title="Schema" />
+
+## Links
+
+- 🎮 [GraphQL Playground](https://p2kwd3i3a8.execute-api.eu-central-1.amazonaws.com/staging/graphql)
+- 🚀 [GraphQL Voyager](https://margarita-graphql-voyager.now.sh/)
+- 🕸️ [Margarita web](http://margarita.kiwi.com)
+- 🥝 [Open source projects at Kiwi.com](https://margarita-graphql-voyager.now.sh/)
```

Comment:
URL points to interactive schema instead of Kiwi.com repo.

Reply:
Thank you! Fixed